Cargo.toml

```
@@ -13,11 +13,13 @@ rayon = ["blake3/rayon"]
 
 [dependencies]
 anyhow = "1.0.69"
-axum = "0.6.9"
+axum = { version = "0.6.9", features = ["multipart"] }
 axum-macros = "0.3.4"
 bao = "0.12.1"
 blake3 = "1.3.3"
 carbonado = "0.3.0-rc.4"
```

```suggestion
carbonado = "0.3.0-rc.7"
```
Let's bump this to rc.7, I published that a little bit ago

---

removing features = ["multipart"]  from axum as we turned direction towards BodyStream.